src/js/components/Intro/IntroNetworkBallotIsNext.jsx

```diff
@@ -15,50 +27,25 @@ export default class IntroNetworkBallotIsNext extends Component {
     this.state = {};
   }
 
-  componentWillMount () {
-    document.body.style.backgroundColor = '#A3A3A3';
-    document.body.className = 'story-view';
-    cookies.setItem('show_full_navigation', '1', Infinity, '/');
-  }
-
-  componentWillUnmount () {
-    document.body.style.backgroundColor = null;
-    document.body.className = '';
-  }
-
   render () {
     renderLog(__filename);
     return (
-      <div
-        className="intro-story intro-story__background background--image5"
-        style={isCordova() ? { backgroundImage: 'url(./img/global/intro-story/slide5-flagpole-698x600.jpg)' } : null}
-      >
-        <Helmet title="See Your Ballot - We Vote" />
-        <div className="intro-story__h1--alt">We Vote</div>
-        <div id="header2" className="intro-story__h2 intro-story__padding-top">
-          Ready to
-          {' '}
-          <strong>vote your values</strong>
-          ?
-        </div>
-        <div id="header2" className="intro-story__h2">
-          On the next screen, you&apos;ll see
-          <br />
-          the next election
-          <br />
-          in your area.
+      <div className="intro-story__padding">
+        <div className="intro-story__h1">Plan your entire ballot<br /><span className="intro-story__h1--highlight">in 6 minutes</span></div>
```

Comment:
Instead of:
<div className="intro-story__h1">Plan your entire ballot<br /><span className="intro-story__h1--highlight">in 6 minutes</span></div>

This should be:
<div className="intro-story__h1">
Plan your entire ballot
<br />
<span className="intro-story__h1--highlight">
in 6 minutes
</span>
</div>